cat-home/src/test/java/com/dianping/cat/report/task/TaskHelperTest.java

```diff
@@ -65,4 +76,84 @@ public void testTodayZero() {
 		Assert.assertEquals(cal.getTimeInMillis(), todayZero.getTime() + 1L);
 	}
 
+    @Test
+    public void testTodayZero2() {
+        Assert.assertEquals(new Date(1562713200000L),
+                TaskHelper.todayZero(null));
+        Assert.assertEquals(new Date(1562713200000L),
+                TaskHelper.todayZero(new Date()));
+    }
+
+    @Test
+    public void testJoin1() {
+
```

Comment:
Should remove this space to keep consistent with the rest or your class

Reply:
done